frontend/src/components/QueueTable.js

```diff
@@ -91,6 +93,13 @@ const QueueTable = observer(queueData => {
           notes: false,
           id: x.id,
         }))}
+        actions={[
+          {
+            icon: "assignmentIndIcon",
+            tooltip: "Edit Participant",
+            onClick: () => history.push("/participantInfo"),
+          },
+        ]}
         columns={[
```

Comment:
This all looks good, but in the form 5 is highest priority, but in the default sorting 1 is highest

Reply:
Could you fix that?